src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,12 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Excavator(),new Bulldozer(),new Truck()};
+
+        for (Machine machine:machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```
be attentive to spaces